src/doc/en/reference/references/index.rst

```diff
@@ -607,6 +607,10 @@ REFERENCES:
               Smith: Faster computation of isogenies of large prime degree.
               ANTS XIV, Open Book Series Vol. 4, No. 1, 2020.
               :arxiv:`2003.10118`
+              
+.. [BDLS2022] Alexander E. Black and Jesús A. De Loera and Niklas Lütjeharms
+              and Raman Sanyal. *The Polyhedral Geometry of Pivot Rules and
+              Monotone Paths* :arxiv: `2201.05134`
```

Comment:
```suggestion
.. [BDLS2022] Alexander E. Black, Jesús A. De Loera, Niklas Lütjeharms,
              and Raman Sanyal. *The Polyhedral Geometry of Pivot Rules and
              Monotone Paths*. :arxiv:`2201.05134`
```

Note this is not a BibTeX entry - so the author list needs to be formatted as it is intended to appear. (and and and is BibTeX syntax)

`:arxiv:...` is syntax of a Sphinx role (see https://deploy-preview-37575--sagemath.netlify.app/html/en/developer/sage_manuals#hyperlinks), no space allowed.